src/main/java/htsjdk/samtools/cram/build/ContainerFactory.java

```diff
@@ -117,9 +118,9 @@ private static Slice buildSlice(final List<CramCompressionRecord> records,
                                     final CompressionHeader header)
             throws IllegalArgumentException, IllegalAccessException,
             IOException {
-        final Map<Integer, ExposedByteArrayOutputStream> map = new HashMap<Integer, ExposedByteArrayOutputStream>();
```

Comment:
Removed the encapsulation-breaking `ExposedByteArrayOutputStream` from Encodings/Codecs.  There are a few remaining instances in CRAM but they will have to wait for future PRs